src/platform/site-wide/sass/modules/_m-homepage.scss

```diff
@@ -167,3 +167,15 @@
     margin-bottom: inherit;
   }
 }
+
+.homepage-image-wrapper {
+  width: 100%;
+  height: 0;
+  padding-bottom: 65%; /* You define this doing height / width * 100% */
+  position: relative;
+  > .lazy {
+    width: 100%;
+    /*height: auto;*/
+    position: absolute;
+  }
+}
```

Comment:
Are these style rules necessary for any elements that are lazy loaded, or just for those particular images on the homepage? Wondering if there's any way we can streamline this in case other pages end up wanting to use it. Probably only a major concern on the homepage either way.

Reply:
These are necessary if you have images that are expanding to fit an area and you want to keep the aspect ratio. The padding-bottom value is image specific, so we can't make it completely generic. Though maybe a mixin would work? I think we could do that in a separate ticket and put it in the design system, once this is in production for a bit and is working alright.

Reply:
Gotcha, I think that's an awesome idea! Also helps to make the strategy more discoverable as we add images.